multiqc/modules/picard/HsMetrics.py

```diff
@@ -108,7 +150,102 @@ def parse_reports(self):
             if s_name not in self.general_stats_data:
                 self.general_stats_data[s_name] = dict()
             self.general_stats_data[s_name].update( data[s_name] )
-    
-    # Return the number of detected samples to the parent module
+        data_table = _clean_table(data)
+        table_html = plots.table.plot(data_table, _get_headers(data_table))
+        if not isinstance(self.sections, list):
+            self.sections = list()
+        self.sections.append({
+                'name': 'HSMetrics',
+                'anchor': 'picard_hsmetrics',
+                'content': table_html})
+        self.sections.append(_add_target_bases(data))
+        self.sections.append(_add_hs_penalty(data))
```

Comment:
Only problem with this is that we still get a section (though empty) if no results:
![image](https://cloud.githubusercontent.com/assets/465550/15932175/fbe6d284-2e5a-11e6-826c-3a83a6f6542d.png)

Maybe instead return `None` and then do:
```python
penalty_html = _add_hs_penalty(data)
if penalty_html is not None:
    self.sections.append(penalty_html)
```
Not as pretty code, but gives a cleaner report..